toSentence.js

```diff
@@ -1,6 +1,18 @@
 var rtrim = require('./rtrim');
 
 module.exports = function toSentence(array, separator, lastSeparator, serial) {
+  // remove empty elements from array
+  for (var i = array.length - 1; i >= 0; i--) {
```

Comment:
Did just see that `filter` is not supported in IE < 9. @epeli what do you think do we need to support IE < 9? IMO not, this can be polyfilled.